scenarios/tracking/02_evaluating_mot_challenge.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<ul><li>Maybe move "The <s>following</s> above results on MOT Challenge training set ...to obtain the results", to after the 2 cells for MOT17 and MOT16 training datasets</li><li>Is it possible to show the results for MOT17/MOT16 test datasets to show they match the results in the table?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Unfortunately, mot challenge didn't provide ground truth for testing data. If want to get the eval results, need to submit to the evaluation server.